ts/Series/ColumnRangeSeries.ts

```diff
@@ -130,6 +134,13 @@ BaseSeries.seriesType<typeof Highcharts.ColumnRangeSeries>('columnrange', 'arear
     (defaultOptions.plotOptions as any).arearange,
     columnRangeOptions
 ), {
+    setOptions: function (this: Highcharts.ColumnRangeSeries): Highcharts.ColumnRangeSeriesOptions {
+        var options = arearangeProto.setOptions.apply(this, arguments);
+        if (options.stacking) {
+            options.stacking = columnRangeOptions.stacking;
```

Comment:
Might as well just delete it, since `columnRangeOptions.stacking` should always be unefined.

Reply:
The `columnRangeOptions.stacking` was still able to be overwritten.  For now this was the workaround, but I will have a look at this to see if there is a better way of doing this.